src/admin/schema/migrate-None-to-0.sql.jinja

```diff
@@ -0,0 +1,26 @@
+CREATE USER 'qsmaster'@'localhost';
+CREATE USER 'qsmaster'@'%';
+
+GRANT ALL ON qservResult.* TO 'qsmaster'@'localhost';
+GRANT ALL ON qservResult.* TO 'qsmaster'@'%';
+
+-- Secondary index database (i.e. objectId/chunkId relation)
+-- created by integration test script/loader for now
+GRANT ALL ON qservMeta.* TO 'qsmaster'@'localhost';
+GRANT ALL ON qservMeta.* TO 'qsmaster'@'%';
+
+-- CSS database
+GRANT ALL ON qservCssData.* TO 'qsmaster'@'localhost';
+GRANT ALL ON qservCssData.* TO 'qsmaster'@'%';
+
+-- TODO FIGURE OUT HOW TO HANDLE THE POD NAME IN CONTAINER-LAND
+-- -- Grant root access to replication controller pod
+-- GRANT ALL ON *.* TO 'root'@'{{ReplicationControllerFQDN}}' IDENTIFIED BY 'CHANGEME' WITH GRANT OPTION;
+
+-- Create user for external monitoring applications
+CREATE USER 'monitor'@'localhost' IDENTIFIED BY {{ mysql_monitor_password }};
+CREATE USER 'monitor'@'%' IDENTIFIED BY {{ mysql_monitor_password }};
```

Comment:
do I? seems to work without. How do I know when I need quotes and when I don't?

Reply:
Manual says password should be a quoted string: https://mariadb.com/kb/en/create-user/
But mysql is notorious for allowing all sorts of broken syntax, so it might work without quotes even if it's not documented.